### PR TITLE
playerctl: update to 0.6.1

### PR DESCRIPTION
**Pull request by abenson**

### CODE REVIEW

File: srcpkgs/playerctl/template [hunk 1]
```diff
@@ -12,7 +12,7 @@ maintainer="Duncaen <duncaen@voidlinux.eu>"
 license="LGPL-3.0-or-later"
 homepage="https://github.com/acrisci/playerctl"
 distfiles="https://github.com/acrisci/playerctl/archive/v${version}.tar.gz"
-checksum=c7af7291fa3efad07dabae62ad31f366e8730a84439e2499c6fb6b09f4a7d0e7
+checksum=5857b0f9c733fecd4a16664d2e401a52a0be29b135b8b2d90b2bb454369ba2ea
 disable_parallel_build=yes
```
Comment (cr6git) [inline srcpkgs/playerctl/template]: I think `disable_parallel_build=yes` is not needed anymore. ping @Duncaen

**Comment by cr6git** [comment]
@abenson this morning I experimented with meson, using this [template](https://gist.github.com/cr6git/05af34284816c74cec00c31a08c01fec).

**Comment by abenson** [comment]
Your meson version works perfectly, even with cross build

**Comment by cr6git** [comment]
@abenson Except for `armv6l-musl` ? https://travis-ci.org/void-linux/void-packages/jobs/409364968#L718

**Comment by cr6git** [comment]
I will not merge this without your approval @Duncaen

File: srcpkgs/playerctl/template [hunk 2]
```diff
@@ -1,30 +1,27 @@
...
 pkgname=playerctl
-version=0.6.0
+version=0.6.1
 revision=1
-build_style=gnu-configure
-configure_args="$(vopt_enable gir introspection)"
-hostmakedepends="automake pkg-config glib-devel gtk-doc
+build_style=meson
+configure_args="$(vopt_if gir '-Dintrospection=true' '-Dintrospection=false')
+ $(vopt_if doc '-Dgtk-doc=true' '-Dgtk-doc=false')"
```
Comment (maxice8) [inline srcpkgs/playerctl/template]: can be replaced with

`-Doption=$(vopt_if true false)`

**Pull request closed without merging**